src/renderer/components/Settings/tabs/MaestroPromptsTab.css

```
@@ -0,0 +1,159 @@
+	margin-bottom: 12px;
+	border-radius: 4px;
+	font-size: 12px;
+}
+
+.prompt-textarea {
+	flex: 1;
+	width: 100%;
+	padding: 12px;
+	border: 1px solid;
+	border-radius: 4px;
+	font-family: 'Monaco', 'Menlo', 'Consolas', monospace;
```

Stylelint (17.6.0)

[error] Unexpected quotes around "Monaco" (font-family-name-quotes)

(font-family-name-quotes)